src/content/blog/sponsor-thanks.md

```diff
@@ -0,0 +1,59 @@
+
+## So many more wonderful sponsors!
+
+Here are some of many sponsors who help out via [GitHub Sponsors](https://github.com/sponsors/ddev).
+
+![GitHub Sponsors](/img/blog/2025/03/ddev-github-sponsors.png)
+
+## In-Kind Sponsors
+
+The open-source community has lots of folks helping out multiple collaborators. In our case, [MacStadium](https://macstadium.com) provides us testing resources, and [JetBrains](https://jetbrains.com) provides a subscription to their IDE products, thanks!
+
+But we never forget Docker. Docker has adopted us in the [Docker-Sponsored-Open-Source (DSOS)](https://docs.docker.com/docker-hub/repos/manage/trusted-content/dsos-program/) program for a few years now, and this means that all of your image pulls are sponsored in that project. You may know that normal image pulls will be strictly limited starting April 1, 2025, but because of the DSOS, users of DDEV won't hit that restriction.
+
+And please don't forget that the [open-source Docker project](https://github.com/moby/moby) underlies every single Docker provider. It doesn't matter if you're using OrbStack or Lima or Docker Desktop or whatever, they're all built on top of the amazing Docker open-source project.
+
+## JSON Detail about Sponsorships
+
+We recently set up a [full updated JSON feed](https://github.com/ddev/sponsorship-data) that can be used to see exactly where DDEV's sponsorship comes from and how we're doing. You can see the latest data any day in the [all sponsorships](https://github.com/ddev/sponsorship-data/blob/main/data/all-sponsorships.json) feed. And [Mark Conroy](https://bsky.app/profile/mark.ie) has built a [web component](https://web-components.mark.ie/web-components/ddev/sponsors-banner/) that can be used to show where we're at on any website. Thank you!
+
+## Many Ways to Sponsor
+
+We have almost too many ways that you can sponsor, but we want to make it easy for you and your organization.
+
+- For individuals and some organizations, [GitHub Sponsors](https://github.com/sponsors/ddev) is super easy, takes moments and can be changed any time.
+- We are happy to invoice your organization and do more of a "support contract" so that you don't have to explain open-source to your finance department. We accept ACH, Wise.com, bank transfer, IBAN, SWIFT, checks, we'll work with your organization to make it work for you. Just send us a note [via the contact page](/contact) or any other way.
+- [PayPal to DDEV](https://www.paypal.com/donate?hosted_button_id=R42QBRNKKSFFL)
+
+## Oops, are you missing or did I forget you?
+
+Is your name or your organization's name missing here? It's easy to get it here. How much does DDEV's support and ongoing maintenance mean to your organization?
+
+(If I made a mistake and left you out let me know and I'll edit this.)
```

Comment:
Passing on a small change suggestion coming from tools that suggest grammar/spelling changes. Add a comma.

(If I made a mistake and left you out, let me know and I’ll edit this.)

Reply:
```suggestion
(If I made a mistake and left you out, let me know and I'll edit this.)
```